### ISSUE TITLE
adapt_framework: Marking aria-label does not read

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#3190]
### Subject of the issue

`.btn__marking`:
* Should be moved in the dom to after the buttons so that it reads in a good order
* The aria label should be moved inside the element so that it reads correctly and the js updated accordingly
```hbs
  <div class="btn__marking js-btn-marking u-display-none">
    <span class="aria-label">{{#all _isInteractionComplete _canShowMarking}}{{#if _isCorrect}}{{@root/_globals._accessibility._ariaLabels.answeredCorrectly}}{{else}}{{@root/_globals._accessibility._ariaLabels.answeredIncorrectly}}{{/if}}{{/all}}</span>
    <div class="icon" aria-hidden="true"></div>
  </div>
```

https://github.com/adaptlearning/adapt_framework/blob/23800a0498563281d0bb5513d71bd3df23a72d4e/src/core/templates/buttons.hbs#L3-L5
https://github.com/adaptlearning/adapt_framework/blob/23800a0498563281d0bb5513d71bd3df23a72d4e/src/core/js/views/buttonsView.js#L170
